new-lamassu-admin/src/components/inputs/Select.styles.js

```diff
@@ -0,0 +1,77 @@
+import { zircon, comet, white, fontSecondary } from '../../styling/variables'
+
+const WIDTH = 152
+
+export default {
+  select: {
+    width: WIDTH,
+    marginBottom: 20,
+    '& button': {
+      position: 'relative',
+      border: 0,
+      backgroundColor: zircon,
+      width: WIDTH,
+      padding: '6px 0 6px 12px',
+      borderRadius: 20,
+      fontFamily: fontSecondary,
+      fontSize: 18,
+      fontWeight: 500,
```

Comment:
Still not what I meant, doesn't this equal some set typography in typography.js? It will probably be some set font style already. like tl2, tl1, or info1... Search for `extend:` on the codebase to see the use